compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -708,6 +708,27 @@ struct RewriteExternCallOpToDynamicImportCallOp
                                          "and does not need an import wrapper");
```

Comment:
we'll want to switch on the existing `wrapAndCallImport` or this new `wrapAndCallLibDeviceImport` based on the `hal.import.static` - we could probably rename both functions to be `wrapAndCallDynamicImport` and `wrapAndCallStaticImport` respectively to make it clearer what they're doing (the old naming was ambiguous).